API.md

```diff
@@ -0,0 +1,1819 @@
+          "lastUpdated": "2020-05-29T00:42:54+00:00"
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/3"
+        },
+        "item": [
+          {
+            "linkId": "0",
+            "text": "cough",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "1",
+            "text": "difficulty-breathing",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "2",
+            "text": "fever",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "3",
+            "text": "used-a-fever-reducer",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "4",
+            "text": "chills",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "5",
+            "text": "repeated-shaking-with-chills",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "6",
+            "text": "muscle-pain",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "7",
+            "text": "headache",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "8",
+            "text": "sore-throat",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "9",
+            "text": "new-loss-of-taste-or-smell",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          }
+        ],
+        "resourceType": "QuestionnaireResponse"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+
+<a name="create"/>
+
+## Creating
+
+The API supports creating new monitorees.
+
+### Extensions
+
+<a name="create-ext"/>
+
+Along with supporting the US Core extensions for race, ethnicity, and birthsex, Sara Alert includes four additional extensions for things specific to the Sara Alert workflows.
+
+Use `http://saraalert.org/preferred_contact_method` for specifying the monitorees Sara Alert preferred contact method (options are: `E-mailed Web Link`, `SMS Texted Weblink`, `Telephone call`, and `SMS Text-message`).
+
+```json
+{
+  "url": "http://saraalert.org/preferred_contact_method",
```

Comment:
We may want to consider this URL, because extensions *should* resolve to a StructureDefinition that describes the extensions.

Will we put a StructureDefinition at these locations? We may want something like `http://saraalert.org/StructureDefinition/preferred_contact_method` or something to be FHIR-like.

We have this problem in Synthea, where we don't actually have our extension definitions available, and it causes some validators to throw ERRORs on our resources.

Reply:
Changed